### DISCUSSION TITLE
How to set default sleected value by only passing ID?

### DISCUSSION THREAD

**Discussion by vitalijalbu** [ant-design/ant-design#43184] [Q&A]
For example I'd have ID = 1 and want to automatically set initialvalue 1, but it displays the id not the label. Anyone can help?

![Screenshot 2023-06-26 alle 07 49 39](https://github.com/ant-design/ant-design/assets/40239902/d7f25150-ee59-408a-8556-a18bd23c99c6)

**Comment by zombieJ** [comment]
`labelInValue` so that you can set `value={{ label: 'some label', value: 123 }}`

**Reply by vitalijalbu** [reply]
My code:

 ```
<Form.Item 
        label="Select warehouse" 
        name="idWarehouse"
        initialValue={19}
      >
        <Select
          name="idWarehouse"
          loading={loading}
          placeholder="Select warehouse"
          options={warehouses}
          labelInValue
          onChange={handleWarehouseChange}
          onClear={handleWarehouseChange}
          allowClear
        />
      </Form.Item>
```
